cmake/FindGoldLinker.cmake

```diff
@@ -1,6 +1,6 @@
 ## Enable Gold linker if available.
 
-option(USE_GOLD "Use Gold linker" ON)
+option(USE_GOLD "Use Gold linker" OFF)
```

Comment:
Note to self: this preserves the original behavior since `./configure` defaults to `USE_GOLD=ON`.